etc/env/example.env

```diff
@@ -326,7 +334,6 @@ VAULT_SERVICE_URL=http://vault:8200
 OPA_VERSION=latest
 OPA_PORT=8181
 OPA_LOG_LEVEL=debug
-CANDIG_OPA_SECRET=my-secret-root-token
 CANDIG_OPA_SECRET_SERVICE=my-secret-service-token
```

Comment:
This OPA token should also not be set from environment variable

Reply:
This one is a bit trickier to fix because it needs a fix in the OPA repo as well: It doesn't really get used for anything, but I can fix it in a separate PR (pair of PRs)